shell/platform/android/test/io/flutter/plugin/platform/PlatformPluginTest.java

```diff
@@ -140,6 +141,60 @@ public void setNavigationBarDividerColor() {
     }
   }
 
+  @Config(sdk = 29)
+  @Test
+  public void setSystemUiMode() {
+    View fakeDecorView = mock(View.class);
+    Window fakeWindow = mock(Window.class);
+    when(fakeWindow.getDecorView()).thenReturn(fakeDecorView);
+    Activity fakeActivity = mock(Activity.class);
+    when(fakeActivity.getWindow()).thenReturn(fakeWindow);
+    PlatformChannel fakePlatformChannel = mock(PlatformChannel.class);
+    PlatformPlugin platformPlugin = new PlatformPlugin(fakeActivity, fakePlatformChannel);
+
+    if (Build.VERSION.SDK_INT >= 28) {
```

Comment:
does it produce an error without this `if` statement?  I thought `@Config` was emulating the API level.

Reply:
I am not sure I was mirroring another test,. This change is to come up to date with that API level. I am not super familiar with these tests, and the original implementation was untested, so I am happy to change this to whatever is appropriate.

Reply:
got it